test/index.js

```diff
@@ -0,0 +1,37 @@
+var scrape = require("readable-proxy").scrape;
+var path = require("path");
+var fs = require("fs");
+var prettyPrint = require("html").prettyPrint;
+var chai = require("chai");
+var chaiAsPromised = require("chai-as-promised");
+chai.should();
+chai.use(chaiAsPromised);
+var expect = chai.expect;
+
+var testPageRoot = path.join(__dirname, "test-pages");
+var testPages = fs.readdirSync(testPageRoot).map(function(dir) {
+  return {
+    dir: dir,
+    source: path.join(testPageRoot, dir, "source.html"),
+    expected: path.join(testPageRoot, dir, "expected.html"),
+  };
+});
+
+describe("Test page", function() {
+  testPages.forEach(function(testPage) {
+    describe(testPage.dir, function() {
+      it("should render as expected", function() {
+        // Allows up to 10 seconds for parsing to complete.
+        // XXX: Scraping is damn slow. Investigate.
+        this.timeout(10000);
```

Comment:
>  I'm not as familiar with server-side JS, but is there a way to create a document from a local HTML file, and then run Readability on that directly?

Indeed. It's a bit strange. I think the main issue is that there is a circular dependency. With this PR, readability depends on readable-proxy and readable-proxy depends on readability (via the `READABILITY_LIB_PATH` env variable).
IMO the first thing to do would be making Mozilla's readability its own NPM package. Is it planned?
If it isn't, I will do it myself, because I will need it very soon (although it'd be completely inappropriate given Mozilla should be the entity releasing readability on NPM :-/ )

Then the tests should be self-contained to this repo (without dep to readable-proxy) via reading files from the file system. Among other things, this will ensure the latest version of readability will be the one tested (and not the one linked from by readable-proxy which might be different)